components/browser/storage-sync/src/test/java/mozilla/components/browser/storage/sync/PlacesBookmarksStorageTest.kt

```diff
@@ -263,5 +275,108 @@ class PlacesBookmarksStorageTest {
+                    syncs = listOf(
+                        SyncInfo(
+                            at = now + 20,
+                            took = 8000,
+                            engines = listOf(
+                                EngineInfo(
+                                    name = "bookmarks",
+                                    at = now + 25,
+                                    took = 6000,
+                                    incoming = null,
+                                    outgoing = listOf(
+                                        OutgoingInfo(
+                                            sent = 10,
+                                            failed = 5
+                                        )
+                                    ),
+                                    failureReason = null,
+                                    validation = ValidationInfo(
+                                        version = 2,
+                                        problems = listOf(
+                                            ProblemInfo(
+                                                name = "missingParents",
+                                                count = 5
+                                            ),
+                                            ProblemInfo(
+                                                name = "missingChildren",
+                                                count = 7
+                                            )
+                                        ),
+                                        failureReason = null
+                                    )
+                                )
+                            ),
+                            failureReason = null
+                        )
+                    ),
+                    events = emptyList()
+                ))
+            }
+
+            override fun sendBookmarksPing() {
+                when (pingCount) {
+                    0 -> {
+                        BookmarksSync.apply {
+                            Assert.assertEquals("xyz789", uid.testGetValue())
+                            Assert.assertEquals(now + 25, startedAt.testGetValue().asSeconds())
+                            Assert.assertEquals(now + 31, finishedAt.testGetValue().asSeconds())
+                            Assert.assertFalse(incoming["applied"].testHasValue())
+                            Assert.assertFalse(incoming["failed_to_apply"].testHasValue())
+                            Assert.assertFalse(incoming["reconciled"].testHasValue())
+                            Assert.assertEquals(10, outgoing["uploaded"].testGetValue())
+                            Assert.assertEquals(5, outgoing["failed_to_upload"].testGetValue())
+                            Assert.assertEquals(1, outgoingBatches.testGetValue())
+                        }
+                    }
+                    else -> Assert.fail()
+                }
+                Pings.bookmarksSync.send()
```

Comment:
By doing this you already clear the stores, so no need to do that in the `resetGlean` :)